### ISSUE TITLE
[Bug] attempt to call field 'GetActivityInfo' (a nil value)

### ISSUE THREAD

**Issue by JimJoyyy** [Gethe/Aurora#75]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Game Client

Retail

### Current Behavior

Whenever hovering over any group in lfg, a wild lua error appears. In normal usage of lfg window it stacks up to thousands rather quickly. puu.sh clip for clarity
https://puu.sh/Ju54b.mp4

### Expected Behavior

well, it definitelly shouldnt spit out a lot of errors thats for sure

### Steps To Reproduce

open premade groups tab and hover over any active group to create the tooltip. checked with just aurora, bugsack and buggrabber (in order to get the error copypastable easly)

### Additional context

```
1x [ADDON_ACTION_BLOCKED] AddOn 'Aurora' tried to call the protected function 'GetPlaystyleString()'.
[string "@!BugGrabber/BugGrabber.lua"]:480: in function <!BugGrabber/BugGrabber.lua:480>
[string "=[C]"]: in function `GetPlaystyleString'
[string "@FrameXML/LFGList.lua"]:3522: in function <FrameXML/LFGList.lua:3511>
[string "=[C]"]: in function `LFGListUtil_SetSearchEntryTooltip'
[string "@FrameXML/LFGList.lua"]:2597: in function <FrameXML/LFGList.lua:2594>
```

```
215x ...ceAurora/Skin/FrameXML/LFGList.lua:132: attempt to call field 'GetActivityInfo' (a nil value)
[string "@Aurora/Skin/FrameXML/LFGList.lua"]:132: in function <...ceAurora/Skin/FrameXML/LFGList.lua:130>
[string "@Aurora/Skin/init.lua"]:179: in function <Aurora/Skin/init.lua:178>
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: in function `LFGListUtil_SetSearchEntryTooltip'
[string "@FrameXML/LFGList.lua"]:2597: in function <FrameXML/LFGList.lua:2594>
[string "=[C]"]: ?

Locals:
tooltip = GameTooltip {
 0 = <userdata>
 SetQuestLogRewardSpell = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 ItemTooltip = Frame {
 }
 SetHyperlink = <function> defined =[C]:-1
 SetAzeriteEssence = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetVoidDepositItem = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetTraitEntry = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetLootCurrency = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetWorldCursor = <function> defined @FrameXML/GameTooltip.lua:952
 SetLootItem = <function> defined =[C]:-1
 GetSpell = <function> defined @FrameXML/GameTooltip.lua:990
 SetQuestLogSpecialItem = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetRecipeReagentItem = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetMerchantItem = <function> defined =[C]:-1
 BottomOverlay = Texture {
 }
 SetSocketedRelic = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 ProcessInfo = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:236
 NineSlice = Frame {
 }
 OnLoad = <function> defined @FrameXML/GameTooltip.lua:927
 SetUnitDebuffByAuraInstanceID = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetBagItem = <function> defined =[C]:-1
 AddLineDataText = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:324
 SetSpellBookItem = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 textLeft1Font = "GameTooltipHeaderText"
 shouldRefreshData = false
 hasMoney = 1
 SetArtifactPowerByID = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 textRight2Font = "GameTooltipText"
 TopOverlay = Texture {
 }
 SetSocketedItem = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetUnitDebuff = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetMountBySpellID = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetToyByItemID = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 shoppingTooltips = <table> {
 }
 SetAzeritePower = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetLFGDungeonShortageReward = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetEnhancedConduit = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 StatusBar = GameTooltipStatusBar {
 }
 SetPossession = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetAzeriteEssenceSlot = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetUnitBuffByAuraInstanceID = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetConduit = <function> defined =[C]:-1
 SetMinimapMouseover = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetQuestLogCurrency = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetRecipeResultItem = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetQuestPartyProgress = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 ProcessLineData = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:303
 SetPvpTalent = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetExistingSocketGem = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetUnit = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetTotem = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetVoidWithdrawalItem = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:444
 SetInfoBackdropStyle = <function> defined @SharedXML/Tooltip/TooltipDataHandler.lua:338
 Show = <function> defined =[C]:-1
 SetPetActio


```

**Comment by Gethe** [comment]
Try the current beta version and see if you still get either of these.

**Comment by Gethe** [comment]
Fixed in fee1e84eac4b5905a9fa7972be918358cdd13d5f
